### ISSUE TITLE
post-merge review of dc2/run2.1 branch into master

### ISSUE THREAD

**Issue by heather999** [LSSTDESC/ImageProcessingPipelines#159]
As [discussed on Slack](https://lsstc.slack.com/archives/CFL9N02MR/p1598981507030700?thread_ts=1598969445.025700&cid=CFL9N02MR)
Comparing the pre-merge (7f262ef) and merge (50764c7815) commits allows us to take a peek at the changes: https://github.com/LSSTDESC/ImageProcessingPipelines/compare/7f262ef...50764c7815
So far all the updates seem to parsl-only - will also take a look at the SRS specific code to be sure nothing was updated by this merge.

**Comment by jchiang87** [comment]
There appear to be commits from Ben going back to September 4, 2019, pre-dating 7f262ef .

**Comment by heather999** [comment]
Yes, I know but `72262ef` is the commit that was on master before the merge. If you look at the compare [linked in this issue](https://github.com/LSSTDESC/ImageProcessingPipelines/compare/7f262ef...50764c7815), it does indeed show those commits going back to Sep 4 2019 - so I believe this allows us to view all the changes.

**Comment by heather999** [comment]
Additionally, I've taken a look at the directories that contain SRS specific files including:
* `config`
* `python`
* `workflows/srs`
I do not see anything that was updated by this parsl merge.  If anyone sees otherwise, please let me know.
